worlds/checksmate/Items.py

```diff
@@ -0,0 +1,119 @@
+    # "Enemy Piece A": CMItemData(4_901_038, ItemClassification.progression),
+    # "Enemy Piece B": CMItemData(4_901_039, ItemClassification.progression),
+    # "Enemy Piece C": CMItemData(4_901_040, ItemClassification.progression),
+    # "Enemy Piece D": CMItemData(4_901_041, ItemClassification.progression),
+    # "Enemy Piece F": CMItemData(4_901_042, ItemClassification.progression),
+    # "Enemy Piece G": CMItemData(4_901_043, ItemClassification.progression),
+    # "Enemy Piece H": CMItemData(4_901_044, ItemClassification.progression),
+    # TODO: implement castling rule & guarantee major piece on that side for Locations
+    # "Play 00 Castle": CMItemData(4_014, ItemClassification.progression),
+    # "Play 000 Castle": CMItemData(4_015, ItemClassification.progression),
+    # TODO: consider breaking passant into individual pawns, or progressive for outer..center pawns
+    # "Play En Passant": CMItemData(4_011, ItemClassification.progression),
+
+    # == Possible pocket implementation ==
+    # "Progressive Pocket Pawn": CMItemData(4_021, ItemClassification.progression, quantity=3, material=90),
+    # "Progressive Pocket Pawn to Piece": CMItemData(4_022, ItemClassification.progression, quantity=3, material=190,
+    #                                               parents=["Progressive Pocket Pawn"]),
+    # "Progressive Pocket Piece Promotion": parents=["Progressive Pocket Pawn to Piece",
+    #          "Progressive Pocket Pawn"]),
+    # == End possible pocket implementation ==
+}
+
+lookup_id_to_name: Dict[int, str] = {data.code: item_name for item_name, data in item_table.items() if data.code}
+
+material_items: Dict[str, CMItemData] = {
+    item: item_data for (item, item_data) in item_table.items() if item_data.material > 0}
+progression_items: Dict[str, CMItemData] = {
+    item: item_data for (item, item_data) in item_table.items() if
+    item_data.classification == ItemClassification.progression}
+useful_items: Dict[str, CMItemData] = {
+    item: item_data for (item, item_data) in item_table.items() if
+    item_data.classification == ItemClassification.useful}
+filler_items: Dict[str, CMItemData] = {
+    item: item_data for (item, item_data) in item_table.items() if
+    item_data.classification == ItemClassification.filler}
+item_name_groups = {
+    # "Pawn": {"Pawn A", "Pawn B", "Pawn C", "Pawn D", "Pawn E", "Pawn F", "Pawn G", "Pawn H"},
+    # "Enemy Pawn": {"Enemy Pawn A", "Enemy Pawn B", "Enemy Pawn C", "Enemy Pawn D",
+    #                "Enemy Pawn E", "Enemy Pawn F", "Enemy Pawn G", "Enemy Pawn H"},
+    # "Enemy Piece": {"Enemy Piece A", "Enemy Piece B", "Enemy Piece C", "Enemy Piece D",
+    #                 "Enemy Piece F", "Enemy Piece G", "Enemy Piece H"},
+    "Chessmen": {"Progressive Pawn", "Progressive Minor Piece", "Progressive Major Piece", "Progressive Consul"},
+}
+
+
+def create_item_with_correct_settings(player: int, name: str) -> Item:
```

Comment:
How is `create_item_with_correct_settings` different from `create_item` in `__init__`? I'm confused what you would need both for